pyproject.toml

```
@@ -0,0 +1,11 @@
+[tool.black]
+line-length = 100
+skip-string-normalization = 1
+
+[tool.isort]
+atomic = true
+line_length = 100
+multi_line_output = 3
```

```suggestion
profile = "black"
```
Ok so `profile="black"` fixes a number of settings including `multi_line_output` (which just happens to be the same as the one explicitly set in this case): https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#custom-configuration